x/oracle/types/asset.go

```diff
@@ -12,3 +17,24 @@ const (
 	BlocksPerYear   = BlocksPerDay * 365
 	MicroUnit       = int64(1e6)
 )
+
+type (
+	// ExchangeRatePrevote -  Defines a structure to store a validator's prevote on the rate of USD in the denom asset
```

Comment:
You missed updating these type's godocs 👍

Reply:
I had updated with the defines a struct to store validator, i think i did not understand what you were asking, can you clarify for me?